packages/css/tabs.css

```diff
@@ -5,6 +5,33 @@
   --dsc-tabs-content-padding: var(--ds-spacing-5);
   --dsc-tabs-content-color: var(--ds-color-neutral-text-default);
   --dsc-tabs-list-border-color: var(--ds-color-neutral-border-subtle);
+
+  &[data-size='sm'] {
+    --dsc-tabs-tab-padding: var(--ds-spacing-2) var(--ds-spacing-4);
+    --dsc-tabs-content-padding: var(--ds-spacing-4);
+  }
+
+  &[data-size='md'] {
+    --dsc-tabs-tab-padding: var(--ds-spacing-3) var(--ds-spacing-5);
+    --dsc-tabs-content-padding: var(--ds-spacing-5);
```

Comment:
Should `md` be removed and just become part of default? ☺️

Reply:
Yes, my bad